mmv1/products/privateca/CaPool.yaml

```diff
@@ -471,6 +471,16 @@ properties:
           in all issued Certificates. If this is false, CRLs will not be published and the corresponding X.509 extension will not
           be written in issued certificates. CRLs will expire 7 days from their creation. However, we will rebuild daily. CRLs are
           also rebuilt shortly after a certificate is revoked.
+      - !ruby/object:Api::Type::Enum
+        name: 'encodingFormat'
+        description: |
+          Specifies the encoding format of each CertificateAuthority's CA
+          certificate and CRLs. If this is omitted, CA certificates and CRLs
+          will be published in PEM.
+        values:
+          - :ENCODING_FORMAT_UNSPECIFIED
```

Comment:
Generally I prefer not to include the `UNSPECIFIED` fields because they aren't useful to users - that is, a Terraform user will never want to provide `UNSPECIFIED`.

Reply:
```suggestion
```